lib/link.js

```diff
@@ -11,8 +11,20 @@ export default class Link extends Component {
 
   static propTypes = {
     children: PropTypes.oneOfType([
-      PropTypes.string,
-      PropTypes.element
+      PropTypes.element,
+      (props, propName) => {
+        const value = props[propName]
+
+        if (Array.isArray(value) && value.length > 1) {
+          return new Error('Warning: Only the first child on <Link> is rendered')
+        }
+
+        if (typeof value === 'string') {
+          warnLink(`Warning: You're using a string directly inside <Link>. This usage has been deprecated. Please add an <a> tag as child of <Link>`)
+        }
+
+        return null
+      }
     ]).isRequired
   }
```

Comment:
proptypes get compiled away right? If so, this is the *perfect* place to add this. Brilliant